tools/UpdatePaths/Scripts/12400_disabler_to_lp12.txt

```diff
@@ -1 +1 @@
-/obj/item/gun/ballistic/automatic/pistol/security : /obj/item/gun/energy/disabler{@OLD}
\ No newline at end of file
+/obj/item/gun/energy/disabler{@OLD} : /obj/item/gun/ballistic/automatic/pistol/security
```

Comment:
This is wrong. Test the update path on a map file, if you get no updates it worked.